### PR TITLE
Created a timeline app and also Posts model

### PR DESCRIPTION
**Pull request by nishithshowri006**

### CODE REVIEW

File: annoyed-alligators/socl_media/apps/timeline/models.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Posts(models.Model):
+    post_title = models.CharField(max_length=40, verbose_name="title")
+    post_content = models.TextField(verbose_name="content")
+    posted_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    post_date_published = models.DateTimeField(auto_now_add=True,
```
Comment (akshgpt7) [inline annoyed-alligators/socl_media/apps/timeline/models.py]: change this name to `post_date_posted`.

File: annoyed-alligators/socl_media/apps/timeline/models.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+                                               verbose_name="Published on")
+    post_last_edited = models.DateTimeField(auto_now=True,
```
Comment (akshgpt7) [inline annoyed-alligators/socl_media/apps/timeline/models.py]: I don't think we need this field in our case. You can remove this.

**Review by akshgpt7 — CHANGES_REQUESTED** [review]

File: annoyed-alligators/socl_media/apps/timeline/models.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Posts(models.Model):
+    post_title = models.CharField(max_length=40, verbose_name="title")
```
Comment (akshgpt7) [inline annoyed-alligators/socl_media/apps/timeline/models.py]: We'll just have content in our posts, so we don't need a title.

**Review by kunalrustagi08 — APPROVED** [review]
Reviewed standard files. No error.

File: annoyed-alligators/socl_media/apps/timeline/models.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Posts(models.Model):
+    post_content = models.TextField(verbose_name="content")
+    posted_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    post_date_posted = models.DateTimeField(auto_now_add=True,
```
Comment (akshgpt7) [inline annoyed-alligators/socl_media/apps/timeline/models.py:10]: The indentation here is wrong. You just have to align it with the opening brace above.

**Comment by akshgpt7** [comment]
Just make a view and url for it, so that you can see if it's working correctly.
Also, if you can please change the name of the app to `feed` instead of `timeline`. It's a bit more elaborate and relevant.

**Pull request closed without merging**